app/helpers/service_helper/textual_summary.rb

```diff
@@ -142,6 +142,10 @@ def textual_retirement_state
     {:label => _("Retirement State"), :value => @record.retirement_state.to_s.capitalize}
   end
 
+  def textual_lifecycle_state
+    {:label => _("State"), :value => @record.lifecycle_state ? @record.lifecycle_state.humanize.capitalize : _("None")}
```

Comment:
Oh, sorry, just noticed this is missing i18n..

Was that intentional? If not, I guess we should have a..

```ruby
{
  Service::STATE_ERROR_PROVISIONING => _("Error in provisioning"),
  Service::STATE_PROVISIONED => _("Provisioned"),
  Service::STATE_PROVISIONING => _("Provisioning"),
}
```

hash somewhere...